### PR TITLE
Improve table's css

### LINKED ISSUE #154: Improve table CSS
The lack of lines or spacing can make tables hard to read.

image

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #154.

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
A little screenshot:

screen shot 2017-03-16 at 23 10 50

File: css/main.scss [hunk 1]
```diff
@@ -51,3 +51,16 @@ $on-laptop:        800px;
 .highlight {
     background-color: #eef;
 }
+
+td, th {
+  border-bottom: 1px solid #eee;
+  border-right: 1px solid #eee;
```
Comment (frewsxcv) [inline css/main.scss]: Nit: This doesn't match the indentation of the previous lines

Comment (frewsxcv) [inline css/main.scss]: lol at that gnarly `@import` indentation above

**Comment by steveklabnik** [comment]
looks like this is a bit different than https://github.com/rust-lang/blog.rust-lang.org/issues/154#issuecomment-287193573, any specific reasons why?

**Comment by GuillaumeGomez** [comment]
@steveklabnik: I can fix it as well as they suggested but I won't from the CSS.

**Comment by GuillaumeGomez** [comment]
So I "rotated" the table. I think it's the closest to what they want if I'm not wrong?

**Comment by steveklabnik** [comment]
Sorry, I mean the CSS changes are different than those ones.

**Comment by lqd** [comment]
(btw it seems the tables are missing the 15px bottom margin that a lot of the other elements have from [base.scss](https://github.com/rust-lang/blog.rust-lang.org/blob/gh-pages/_sass/_base.scss#L27-L35)  — + tiny typo there with "vertycal" — which is making the paragraph following them "sticking" to the table without spacing :)

**Comment by GuillaumeGomez** [comment]
Updated with @lqd's remarks.

**Comment by steveklabnik** [comment]
Let's :shipit:

**Pull request merged**